src/test/java/org/symphonyoss/symphony/entityjson/TestValidSchemas.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
No assertions at all in this file:(.

Reply:
These methods throw exceptions if the test fails, there is no need for assertions.